examples/spiffs/spiffs_example.c

```diff
@@ -77,6 +85,8 @@ static void example_fs_info()
 void test_task(void *pvParameters)
 {
     esp_spiffs_init();
+    // esp_spiffs_init(0x200000, 0x1000);   // for run-time configuration
```

Comment:
I thought about doing it at first but for some reason rejected it. 
I will surround it with the preprocessor condition.